pyannote/audio/applications/change_detection.py

```diff
@@ -0,0 +1,411 @@
+        <experiment_dir>/config.yml, that describes the architecture of the neural
+        network used for sequence labeling (0 vs. 1, non-speech vs. speech), the
+        feature extraction process (e.g. MFCCs) and the sequence generator used for
+        both training and testing.
+
+        ................... <experiment_dir>/config.yml ...................
+        feature_extraction:
+             name: YaafeMFCC
+             params:
+                    e: False                   # this experiments relies
+                    De: True                   # on 11 MFCC coefficients
+                    DDe: True                  # with 1st and 2nd derivatives
+                    D: True                    # without energy, but with
+                    DD: True                   # energy derivatives
+                    stack: 3
+ 
+        architecture:
+             name: StackedLSTM
+             params:                         # this experiments relies
+                 n_classes: 1                # on one LSTM layer (16 outputs)
+                 lstm: [16]                  # and one dense layer.
+                 mlp: [16]                   # LSTM is bidirectional
+                 bidirectional: 'concat'
+                 final_activation: 'sigmoid'
+
+        type:
+                name: type2
+                params:
+                    num_mfcc: 3
+
+        sequences:
+             duration: 3.2                 # this experiments relies
+             step: 0.8                     # on sliding windows of 3.2s
+             balance: 0.05                 # with a step of 0.8s
+        ...................................................................
+
+"train" mode:
+        First, one should train the raw sequence labeling neural network using
+        "train" mode. This will create the following directory that contains
+        the pre-trained neural network weights after each epoch:
+                <experiment_dir>/train/<database.task.protocol>.<subset>
+
+        This means that the network was trained on the <subset> subset of the
+        <database.task.protocol> protocol. By default, <subset> is "train".
+        This directory is called <train_dir> in the subsequent "tune" mode.
+
+"evaluate" mode:
+        Then, one can evaluate the model with "evaluate" mode. This will create 
+        the following directory that contains coverages and purities based 
+        on different thresholds:
+                <train_dir>/evaluate/<database.task.protocol>.<subset>
+        This means that the model is evaluated on the <subset> subset of the
+        <database.task.protocol> protocol. By default, <subset> is "development".
+
+"apply" mode
+        Finally, one can apply speaker change detection using "segment" mode.
+        This will create the following files that contains the segmentation results:
+                <tune_dir>/segments/<database.task.protocol>.<subset>/threshold/{uri}.0.seg
```

Comment:
Is `threshold` a placeholder here? 
If so, it should be replaced by `{threshold}` and a short sentence could explain that several subdirectories are created, depending on the value of the threshold.